### PR TITLE
[Exec](join) Support column string64 to avoid join failed in string size overflow the uint32

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Before:
join right table column string over uint32
`string column Length is too Large`

After:
get the right result, but may exec slowly

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

File: be/src/vec/functions/in.h [hunk 1]
```diff
@@ -49,11 +50,7 @@
 #include "vec/data_types/data_type_number.h"
 #include "vec/functions/function.h"
 
-namespace doris {
-namespace vectorized {
-class ColumnString;
-} // namespace vectorized
-} // namespace doris
+namespace doris {} // namespace doris
```
Comment (BiteTheDDDDt) [inline be/src/vec/functions/in.h:53]: maybe we can remove this

**Comment by HappenLee** [comment]
run buildall

File: be/src/vec/columns/column_string.cpp [hunk 2]
```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 // This file is copied from
-// https://github.com/ClickHouse/ClickHouse/blob/master/src/Columns/ColumnString.cpp
+// https://github.com/ClickHouse/ClickHouse/blob/master/src/Columns/ColumnStr<T>.cpp
```
Comment (BiteTheDDDDt) [inline be/src/vec/columns/column_string.cpp:18]: does ck have this file?

**Comment by HappenLee** [comment]
run buildall

**Review by BiteTheDDDDt — APPROVED** [review]

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**